Remotion/ObjectBinding/Web/UI/Controls/Validation/BocListValidationResultDispatchingValidator.cs

```diff
@@ -102,8 +112,27 @@ private void DispatchToValidationFailureRepository (IBusinessObjectValidationRes
 
     protected override bool EvaluateIsValid ()
     {
-      // TODO RM-6056: Shows a validation error if IBusinessObjectValidationResult returned messages for this control.
-      return true;
+      var sb = new StringBuilder();
+
+      var bocList = GetControlToValidate();
+      var validationFailureRepository = bocList.ValidationFailureRepository;
+
+      foreach (var validationFailure in validationFailureRepository.GetUnhandledValidationFailuresForBocList(true))
+      {
+        sb.AppendLine(validationFailure.Failure.ErrorMessage);
+      }
+
+      var rowObjects = bocList.Value ?? Array.Empty<IBusinessObject>();
+      var rowsHaveValidationFailures = rowObjects.Any(r => validationFailureRepository.HasValidationFailuresForDataRow(r));
+
+      if (rowsHaveValidationFailures)
+      {
+        sb.Append("BocList has validation failures.");
```

Comment:
this is okay because we know that we will change this in the next iteration. Otherwise, we'd need a resource string. PLease add a todo comment to document his and we should prefix the string with "###" to make it obivous in the UI, too. Once we create a build and it gets used, this will get noticed and reported as a missing internationalization support. By adding a marker (we usually have "###") it is more obvious that his actually intended.